cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/EnergyConsumerConversion.java

```diff
@@ -40,32 +40,46 @@ public void convert() {
     }
 
     private static void setLoadDetail(String type, Load load) {
-        if (type.endsWith("#ConformLoad")) { // ConformLoad represent loads that follow a daily load change pattern where the pattern can be used to scale the load with a system load
-            load.newExtension(LoadDetailAdder.class)
-                    .withFixedActivePower(0)
-                    .withFixedReactivePower(0)
-                    .withVariableActivePower((float) load.getP0())
-                    .withVariableReactivePower((float) load.getQ0())
-                    .add();
-        } else if (type.endsWith("#NonConformLoad")) { // does not participate in scaling
-            load.newExtension(LoadDetailAdder.class)
-                    .withFixedActivePower((float) load.getP0())
-                    .withFixedReactivePower((float) load.getQ0())
-                    .withVariableActivePower(0)
-                    .withVariableReactivePower(0)
-                    .add();
+        float p0 = (float) load.getP0();
```

Comment:
I think that we have to try to remove the use of floats in the extension `LoadDetail`.

Reply:
I agree. I will make an issue and we'll make another PR for this.